internal/controller/controllers/agent_controller_test.go

```diff
@@ -1243,6 +1245,259 @@ var _ = Describe("agent reconcile", func() {
 
 })
 
+var _ = Describe("Approve CSRs", func() {
+	pemCSR := `-----BEGIN CERTIFICATE REQUEST-----
```

Comment:
Can you write a comment of what the decoded certificate should look like?

Reply:
OK

Reply:
Certificate or CSR ?